crates/ruma-client-api/src/error.rs

```diff
@@ -312,7 +312,6 @@ pub enum ErrorBody {
     Json(JsonValue),
 
     /// A response body that is not valid JSON.
-    #[non_exhaustive]
```

Comment:
Shouldn't `MatrixErrorBody`'s variant become exhaustive too? I don't see why one would be exhaustive and not the other.

Reply:
Made this change in a second commit.